core-feature-pack/ee-8-api/pom.xml

```diff
@@ -32,6 +32,7 @@
 
     <artifactId>wildfly-core-feature-pack-ee-8-api</artifactId>
     <packaging>pom</packaging>
+    <name>WildFly Core: Feature Pack for EE 8 API</name>
```

Comment:
This is more "EE 8 API resources for Galleon Feature Packs"

Reply:
fixed, thanks